.github/workflows/actions.yml

```
@@ -0,0 +1,90 @@
+name: quicknxs_ci
+
+env:
+  TRIGGER_URL: "https://code.ornl.gov/api/v4/projects/11852/trigger/pipeline"
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [next]
+    tags: ['v*']
+
+jobs:
+
+  linux:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ["3.8"]
```

```suggestion
    runs-on: ubuntu-20.04
    defaults:
      run:
        shell: bash -l {0}
```

We don't need to use a matrix here since we only have one build. Also `python-version` was not used.